components/pricing/PricingTable.tsx

```diff
@@ -0,0 +1,303 @@
+import { Row, Col, Table } from "react-bootstrap";
+import Image from "../Image";
+export const PricingTable = () => {
```

Comment:
Nitpick of nitpicks 😂 
```suggestion
import Image from "../Image";

export const PricingTable = () => {
```